web/src/click_handlers.js

```diff
@@ -384,16 +384,15 @@ export function initialize() {
         message_edit.toggle_resolve_topic(message_id, topic_name);
     });
 
-    // TOPIC MUTING
-    $("body").on("click", ".message_header .on_hover_topic_mute", (e) => {
-        e.stopPropagation();
-        muted_topics_ui.mute_or_unmute_topic($(e.target), true);
-    });
-
-    $("body").on("click", ".message_header .on_hover_topic_unmute", (e) => {
-        e.stopPropagation();
-        muted_topics_ui.mute_or_unmute_topic($(e.target), false);
-    });
+    // TOPIC MUTING/UNMUTING
+    $("body").on(
+        "click",
+        ".message_header .on_hover_topic_unmute,.message_header .on_hover_topic_mute",
+        (e) => {
+            e.stopPropagation();
+            muted_topics_ui.mute_or_unmute_topic($(e.target));
+        },
+    );
```

Comment:
I'd prefer to not make this change -- it's a lot easier to reason about code that explicitly makes the change that's requested, rather than something trying to do a toggle.